### ISSUE TITLE
Is there a way to launch a game in FULLSCREEN mode?

### ISSUE THREAD

**Issue by gachis** [kitao/pyxel#234]
I've tried what was discussed on #130 but it doesn't work. 
I was wondering if there is a special command to launch on `__init__` to use that `_toggle_fullscreen()`

**Comment by alderven** [comment]
I'm using [keyboard](https://pypi.org/project/keyboard/) lib as a workaround for Windows platform:

```
import pyxel
import keyboard

class App:
    def __init__(self):
        pyxel.init(160, 120)
        self.x = 1
        pyxel.run(self.update, self.draw)

    def update(self):
        keyboard.press_and_release('alt+enter')
        self.x = (self.x + 1) % pyxel.width

    def draw(self):
        pyxel.cls(0)
        pyxel.rect(self.x, 0, 8, 8, 9)

App()
```

**Comment by kitao** [comment]
I'm planning to add fullscreen option for the init API. Please wait.

**Comment by kitao** [comment]
I've released 1.3.2 supports FULLSCREEN startup option.
